### PR TITLE
Adding orocos-kdl v1.3.2

### PR DESCRIPTION
**Pull request by seanyen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by seanyen** [comment]
cc @wolfv

**Comment by wolfv** [comment]
nice! feel free to add me as a maintainer :)

File: recipes/orocos_kdl/bldpkg.bat [hunk 1]
```diff
@@ -0,0 +1,23 @@
+:: MSVC is preferred.
+set CC=cl.exe
+set CXX=cl.exe
```
Comment (scopatz) [inline recipes/orocos_kdl/bldpkg.bat]: This doesn't seem correct

Comment (seanyen) [reply recipes/orocos_kdl/bldpkg.bat]: I think the feedback came in another PR too. Here is the comments: https://github.com/conda-forge/staged-recipes/pull/10985#discussion_r387797675

Just wondering should I fall back to use "NMake Makefiles" generator? (It is fine, however which doesn't make use of the multi-core, so it generally slows the build.) Or any other guidance ?

Comment (scopatz) [reply recipes/orocos_kdl/bldpkg.bat]: CC @conda-forge/core

**Comment by seanyen** [comment]
@conda-forge/staged-recipes This is ready for review and merge. Thanks! (The Azure Pipelines Windows doesn't build because of an infrastructure error, but the AppVeyor one does the proof-building.)

**Pull request merged**